client/src/App.css

```
@@ -0,0 +1,3 @@
+body{
+  background-color:black !important;
```

The background color is not black, and why do you use `!important`? 
```suggestion
  background-color: #1C1C28;
```